content/applications/services/helpdesk/overview/help_center.rst

```diff
@@ -0,0 +1,431 @@
+
+When Odoo reveals the desired article, click it, or highlight the :guilabel:`Article` title, and
+press :command:`Enter`. This will open the article in the :guilabel:`Knowledge` application.
+
+To open the article in a new tab, press :command:`Ctrl + Enter`.
+
+.. tip::
+   If a more in-depth search is required, press :command:`Alt + B`. That will reveal a separate
+   page, in which a more detailed search can occur.
+
+Share articles to the help center
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order for a *Knowledge* article to be available to customers and website visitors, it has to be
+published.
+
+.. note::
+   Even though the *Help* article has been enabled on a team, Odoo will not share all the nested
+   articles to the web. Individual articles intended for customers *must* be published for them to
+   be viewable on the website.
+
+To publish an article, navigate to the desired article, (by following the above steps), and click
+:guilabel:`Share` icon in the upper-right corner. This will reveal a menu. Slide the toggle button
+labeled :guilabel:`Share to Web` to read :guilabel:`Article Published`.
+
+.. image:: help_center/help-center-knowledge-sharing.png
+   :align: center
+   :alt: View of a knowledge article focused on sharing and publishing options.
+
+Solve tickets with templates
+----------------------------
+
+:guilabel:`Template` boxes can be added to *Knowledge* articles to allow content to be reused,
+copied, sent as messages, or added to the description on a ticket. This allows teams to maintain
+consistency when answering customer tickets, and minimize the amount of time spent on responding to
+repeat questions.
+
+Add templates to articles
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To create a :guilabel:`Template`, go to the *Help* article (:menuselection:`Dashboard --> Knowledge
+--> Help`). Click on an existing nested article or create a new one by clicking the :guilabel:`+
+(plus sign)` icon next to *Help*.
+
+Type `/` to open the :guilabel:`Powerbox` and view a list of :doc:`commands
+</applications/productivity/knowledge/articles_editing>`. Select or type `template`. A gray template
+block will be added to the page. Add any necessary content to this block.
+
+.. image:: help_center/help-center-knowledge-template-options.png
+   :align: center
+   :alt: View of a template in knowledge with focus on send and copy options.
+
+.. note::
+   :guilabel:`Templates` will only display the :guilabel:`Use as description` or :guilabel:`Send as
+   Message` options if they are accessed from the :guilabel:`Helpdesk`.
+
+Use templates in tickets
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Templates can be used to respond directly to a *Helpdesk* ticket as a message, or to add information
+to the ticket's description.
+
+To use templates in a *Helpdesk* ticket, first, open a ticket, either from the :guilabel:`Helpdesk`
+dashboard or by going to :menuselection:`Helpdesk --> Tickets --> All Tickets` and selecting a
+ticket from the list.
```

Comment:
How come in this section, _template_ and _templates_ are lowercase, with no guis?
How come the word _ticket_ is not capitalized with a gui like it was in previous sections?